src/clib/pio_file.c

```diff
@@ -109,6 +109,7 @@ int PIOc_createfile(const int iosysid, int *ncidp, int *iotype,
         return PIO_ENOMEM;
 
     /* Fill in some file values. */
+    file->fh = -1;
```

Comment:
In C we don't initialize unless there is a need.

What you are doing is communicating that you are going to depend on that value being -1. But we don't. So don't communicate that we are going to care about that value, since we don't.

I will agree that it should be removed from the log statement on line 191. There's no way it can be set by then, so it's pointless to log it.